playwright-tests/playwright.config.ts

```diff
@@ -0,0 +1,81 @@
+import {PlaywrightTestConfig, devices} from '@playwright/test';
```

Comment:
Feel free to tweak this config as needed. A lot of it is copied over from the default examples in the docs.

https://playwright.dev/docs/test-configuration